test/e2e/v1/e2e_v1_test.go

```diff
@@ -14,12 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
> test/e2e/v1/e2e_v1_test.go

nit: `v1` appears twice.
We probably can change it to `test/e2e/v1/e2e_test.go` to reduce duplicated information in the path :)

Reply:
👍